src/domain/downloadImage.ts

```diff
@@ -0,0 +1,33 @@
+export default async function downloadImage(dataURI: string, filename: string) {
```

Comment:
Hi! downloadImage seems to be in the lib, not in the domain

More about the differences:
1. lib is abstract utilities and infrastructure code, not directly related to GMX, this function fits here

2) Domain - product/business logic, i.e. processing specific user scenarios in GMX context (there should be code about tokens, positions, orders, various calculations etc.)

Reply:
Sure, done!